collections/misc/collprofiles.php

```diff
@@ -389,6 +389,9 @@ function directSubmitAction(e) {
 										<a href="#" onclick="$('li.traitItem').show(); return false;">
 											<?= $LANG['TRAIT_CODING_TOOLS'] ?>
 										</a>
+										<a href="#" onclick="$('li.traitItem').show(); return false;"> 
```

Comment:
We're trying to decouple the codebase from using jQuery. I just noticed that that's not in the coding standards, so I just added it. Sorry about that! I would maybe break out the onclick call to a javaScript function defined in the <script > element of collections/misc/collprofiles.php (maybe after `directSubmitAction` is defined?).